Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM fedora:20
+FROM centos:7
 MAINTAINER Marek Goldmann <mgoldman@redhat.com>
 
 # Execute system update
-RUN yum -y update && yum clean all
+RUN yum -y update && rm -rf /var/cache/yum
```

Comment:
Why not? ;) Probably does the same thing tbh, just wanted to make sure that everything was cleaned up properly.